r/R/dplyr-funcs-datetime.R

```diff
@@ -376,7 +382,7 @@ register_bindings_datetime_conversion <- function() {
         options = list(format = format, unit = 0L, error_is_null = TRUE)
       )
     }
-    output <- build_expr("cast", x, options = cast_options(to_type = timestamp()))
+    output <- build_expr("cast", x, options = cast_options(to_type = timestamp(unit = "ns")))
```

Comment:
For `base::strptime()`, we added a `unit` argument that is not present in the base R function. Can we add that here too, so that we aren't hard-coding nanoseconds?

Reply:
Thank you for letting me know. That makes sense and I will take a closer look.